lib/phoenix/live_dashboard/live/processes_live.ex

```diff
@@ -35,6 +35,16 @@ defmodule Phoenix.LiveDashboard.ProcessesLive do
     <div class="processes-page">
       <h5 class="card-title">Processes</h5>
 
+      <div class="processes-search">
+        <form phx-submit="search" class="form-inline">
+          <div class="form-row align-items-center">
+            <div class="col-auto">
+              <input type="text" name="search" class="form-control" value="<%= @params.search %>" placeholder="Search">
```

Comment:
```suggestion
              <input type="text" name="search" class="form-control" value="<%= @params.search %>" placeholder="Search name or initial call">
```

Reply:
Yeah I wasn't sure, an accurate description would be "Search PID, name or initial call, or current function" but that's way too long. I'm fine with your suggestion but please verify you still wanna go with it :)